File: leo2-node/src/main/java/org/deku/leo2/node/data/repositories/custom/RouteRepositoryCustom.java
package org.deku.leo2.node.data.repositories.custom;

import org.deku.leo2.node.data.entities.Route;
import org.deku.leo2.node.data.entities.RoutePK;

/**
 * Created by JT on 13.05.15.
 */
public interface RouteRepositoryCustom {

    public Route findActualRoute(RoutePK routePK);

}

File: leoz-node/src/main/java/org/deku/leoz/node/data/repositories/master/custom/RouteRepositoryCustom.kt
package org.deku.leoz.node.data.repositories.master.custom

import org.deku.leoz.node.data.repositories.master.RouteRepository
import javax.inject.Inject

/**
 * Created by JT on 13.05.15.
 */
interface RouteRepositoryCustom

/**
 * Created by JT on 13.05.15.
 */
class RouteRepositoryImpl : RouteRepositoryCustom {
    @Inject
    internal var mRouteRepository: RouteRepository? = null
}
